Cargo.toml

```diff
@@ -41,14 +45,29 @@ async-std = "*"
 futures = "*"
 nitrokey = { version = "0.9.0", optional = true }
 once_cell = "1.14.0"
-windows = { version = "0.56.0", features = ["Win32_Security_Tpm", "Security_Cryptography_Core", "Win32_Security_Cryptography"], optional = true }
+windows = { version = "0.56.0", features = [
+    "Win32_Security_Tpm",
+    "Security_Cryptography_Core",
+    "Win32_Security_Cryptography",
+], optional = true }
 tss-esapi = { version = "7.5.0", optional = true }
 serde = { version = "*", features = ["derive"] }
 serde_json = "1.0.109"
 tracing = { version = "0.1.40", features = ["std", "log"] }
 tracing-subscriber = "0.3.18"
 tracing-appender = "0.2.3"
 yubikey = { version = "0.8.0", optional = true }
+reqwest = { version = "0.12.4", features = ["json"] }
+tokio = { version = "1", features = ["full"] }
+openssl = "0.10.64"
+base64 = "0.22.1"
+ed25519-dalek = "2.1.1"
+arrayref = "0.3.7"
+sodiumoxide = "0.2.7"
```

Comment:
Sodiumoxide is deprecated. Please see https://rustsec.org/advisories/RUSTSEC-2021-0137. Please investigate the possibility of using an alternative.

Reply:
fixed in [6611515](https://github.com/cep-sose2024/rheinsec_rust-crypto/commit/6611515c707ebae1a6977886ec864955be7caf97)